asdf/yamlutil.py

```diff
@@ -318,11 +324,11 @@ def dump_tree(tree, fd, ctx):
 
     ctx : Context
         The writing context.
-    """
-    class AsdfDumperTmp(AsdfDumper):
-        pass
-    AsdfDumperTmp.ctx = ctx
 
+    tree_finalizer : callable, optional
+        Callback that receives the tagged tree before it is validated
+        and defaults are removed.
+    """
```

Comment:
Missing the description for the _serialization_context arguement.

Reply:
I omitted this intentionally, since it's not part of the public API.  I didn't want to advertise it to external developers who might be using the method, since it's going to disappear in 3.0 anyway.  Bad idea?

Reply:
OK, perhaps a comment after the docstring should clarify the intent